test/fuzz/tests/mempool_test.go

```diff
@@ -33,11 +34,11 @@ func init() {
 	}
 }
 
-func Fuzz(data []byte) int {
-	err := getMp().CheckTx(context.Background(), data, nil, mempool.TxInfo{})
-	if err != nil {
-		return 0
-	}
-
-	return 1
+func FuzzMempool(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data []byte) {
```

Comment:
@elias-orijtech I don't think they are using Go1.18 yet.

Reply:
Good catch. I've added build tags to match Go 1.18 and the OSS-Fuzz builder.